### ISSUE TITLE
Rename default config path from .conf -> .toml

### ISSUE THREAD

**Issue by Teascade** [Teascade/pipeswitch#9]

**Comment by strohel** [comment]
This would be nice, as currently there's a confusing situation where README mentions .toml, but code still uses .conf.

**Comment by tkna91** [comment]
Yes, with the current `pipeswitchd-bin 1.0.2-1`, `pipeswitch.conf` is automatically put and loaded when `pipeswitchd` starts up.
`pipeswitch.toml` is not used.
